client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts

```diff
@@ -267,11 +269,11 @@ export const useInboundShipmentColumns = () => {
       getRowExpandColumn(),
       GenericColumnKey.Selection,
     ],
-    { sortBy, onChangeSortBy },
-    [sortBy, onChangeSortBy]
+    { sortBy, onChangeSortBy: updateSortQuery },
+    [sortBy, updateSortQuery]
   );
 
-  return { columns, onChangeSortBy, sortBy };
+  return { columns, onChangeSortBy: updateSortQuery, sortBy };
```

Comment:
I can get rid of the `onChangeSortBy` but it seems the `sortBy` is being used extensively in its parent.

Reply:
oh ok - it was more that the parent could fetch the `sortBy` from `useUrlQueryParams` rather than via the columns - but I see that the parent isn't using the query params hook - so all good as you are.